src/main/java/org/opentripplanner/street/model/edge/VehicleParkingEdge.java

```diff
@@ -92,22 +92,33 @@ protected State[] traverseUnPark(State s0) {
 
     if (streetMode.includesBiking()) {
       final BikePreferences bike = s0.getPreferences().bike();
-      return traverseUnPark(s0, bike.parkCost(), bike.parkTime(), TraverseMode.BICYCLE);
+      return traverseUnPark(
+        s0,
+        bike.parking().parkCost().toSeconds(),
+        (int) bike.parking().parkTime().toSeconds(),
+        TraverseMode.BICYCLE
+      );
     } else if (streetMode.includesDriving()) {
       final CarPreferences car = s0.getPreferences().car();
-      return traverseUnPark(s0, car.parkCost(), car.parkTime(), TraverseMode.CAR);
+      return traverseUnPark(
+        s0,
+        car.parking().parkCost().toSeconds(),
```

Comment:
If the `traverseUnPark` method would accept a `Duration` you'd have to convert to int only once.

Reply:
Not sure, I'll check

Reply:
This is now done